workers/event_catcher/event_catcher.rb

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env ruby
+
+require "manageiq-messaging"
+require "pathname"
+require "sd_notify"
```

Comment:
Minor but these are auto-required when run through bundle exec so these lines technically aren't needed

Reply:
Actually they're not?  `SdNotify`, `ManageIQ::Messaging`, and `RbVmomi` all fail without these requires.  Not sure what's up there yet (this is running with bundle exec)